packages/support/lib/net.js

```diff
@@ -84,6 +83,7 @@ async function uploadFileToHttp(
         }
       }
     }
+    form.append(fileFieldName, localFileStream);
```

Comment:
Please add the explanation comment here, so this sequence won't be accidentally changed in the future

Reply:
Added comment "AWS S3 POST upload requires this to be the last field"